painttiming.bs

```diff
@@ -31,11 +31,16 @@ urlPrefix: https://www.w3.org/TR/CSS2/visuren.html; spec: CSS-2;
 
 Introduction {#intro}
 =====================
+
+<div class=non-normative>
+
+<em>This section is non-normative.</em>
+
 Load is not a single moment in time — it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast" or "slow".
 
-First paint (FP) is the first of these key moments, followed by first contentful paint (FCP). These metrics mark the points, immediately after navigation, when the browser renders pixels to the screen. This is important to the user because it answers the question: is it happening?
+First paint (FP) is the first of these key moments, followed by first contentful paint (FCP). These metrics mark the points in time when the browser renders pixels to the screen. This is important to the user because it answers the question: is it happening?
```

Comment:
"when the browser renders pixels to the screen" is a misleading statement since we're measuring the time in the event loop's thread, and not the painting / composing thread, or even the time in the window sever which pushes frames to GPU or even time in GPU which pushes a frame buffer to the screen. I'd rephrase it as "when the browser renders a given document".